pages/bb/verveprogram.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Verveprogram
+---
+
+# Verveprogram
+
+Liker du Bare Bitcoin og vil anbefale oss til andre? 
+Det er supre nyheter, for gjennom invitasjonsprogrammet vårt kan både 
+du og den du inviterer din få bitcoin som takk for hjelpen.
+
+## Hvordan funker det?
+
+1. Finn QR-kode og invitasjonskoden din i inviter fanen, denne ser du ved å trykke på 
+''Se Invitasjonskoden min''
```

Comment:
Burde heller bruke `"`, i stedet for `''` som du har brukt her (vanlig hermetegn, ikke to enkle tegn etter hverandre)